floss/main.py

```diff
@@ -546,6 +542,12 @@ def main(argv=None) -> int:
         results.strings.static_strings = static_strings
         results.metadata.runtime.static_strings = get_runtime_diff(interim)
         interim = time()
+    else:
+        with open(sample, "rb") as f:
+            with contextlib.closing(mmap.mmap(f.fileno(), 0, access=mmap.ACCESS_READ)) as buf:
+                static_strings = list(extract_ascii_unicode_strings(buf, args.min_length))
+
+    language = identify_language(sample=sample, static_strings=static_strings)
```

Comment:
if we need these now anyway, let's refactor the logic to always extract and check the language first

then, if static strings are enabled, we can also add them to the results data